lib/lockstep_sdk/lockstep_api.rb

```diff
@@ -169,6 +169,9 @@ def request(method, path, body, params)
             request["Content-Type"] = 'application/*+json'
             request.body = body
             
+            request["SdkType"] = 'Ruby'
+            request["SdkVersion"] = '2022.3.50.0'
```

Comment:
As long as we're adding SdkType and SdkVersion, can we please also add MachineName and ApplicationName?

Looks like we can do it via:

```
require 'socket'
hostname = Socket.gethostname
```

https://stackoverflow.com/questions/42566/getting-the-hostname-or-ip-in-ruby-on-rails

Reply:
Done!